docs/development-local.md

```diff
@@ -0,0 +1,34 @@
+We assume that at this point you've at the very least cloned the repo locally and have it open in your favorite editor. If you haven't made it there, please take a moment to get yourself set up and comfortable. 
+
+
+## Build BitOps-base
```

Comment:
they're building a custom image, not base. maybe change to "Build your custom BitOps image"